app/uk/gov/hmrc/gform/sharedmodel/formtemplate/FormComponent.scala

```diff
@@ -214,6 +214,10 @@ object IsSummarySection {
   def unapply(fc: FormComponent): Option[SummarySection] = fc.`type`.cast[SummarySection]
 }
 
+object IsDeclarationSection {
+  def unapply(fc: FormComponent): Option[DeclarationSection] = fc.`type`.cast[DeclarationSection]
```

Comment:
`ComponentType` has no subtype `DeclarationSection` . How can this ever succeed?

What am I missing?

Reply:
Sorry It's my bad. Fixed